### PR TITLE
Grid Toolbar Update

### PR DESCRIPTION
**Pull request by IGSvt**
-Reference image update
-Adding info regarding Figma

### CODE REVIEW

File: en/images/grid_toolbar_demo.png
Comment (imincheva) [inline en/images/grid_toolbar_demo.png:1]: There's a transparent stripe between the grid and the toolbar.

File: en/images/grid_toolbar_demo@2x.png
Comment (imincheva) [inline en/images/grid_toolbar_demo@2x.png:1]: There's a transparent stripe between the grid and the toolbar.

File: en/images/grid_toolbar_styling.png
Comment (imincheva) [inline en/images/grid_toolbar_styling.png:1]: There's a transparent stripe between the grid and the toolbar.

File: en/images/grid_toolbar_styling@2x.png
Comment (imincheva) [inline en/images/grid_toolbar_styling@2x.png:1]: There's a transparent stripe between the grid and the toolbar.

File: en/images/grid_toolbar_layout.png
Comment (imincheva) [inline en/images/grid_toolbar_layout.png:1]: There's empty space on the image, which is transparent.

File: en/images/grid_toolbar_layout@2x.png
Comment (imincheva) [inline en/images/grid_toolbar_layout@2x.png:1]: There's empty space on the image, which is transparent.

**Review by imincheva — CHANGES_REQUESTED** [review]
Images need to be updated.

File: en/images/grid_toolbar_layout.png
Comment (IGSvt) [reply en/images/grid_toolbar_layout.png:1]: thats on purpose so the image has same size as all the references

Comment (IGSvt) [reply en/images/grid_toolbar_layout.png:1]: I have changed it however this is done so the images used for reference appear with the same scale

**Review by imincheva — DISMISSED** [review]

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**